rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -68,12 +144,49 @@ components:
     Accepted:
       description: The transformation request was accepted for processing.
       headers:
-        'Content-Version':
+        Content-Version:
+          $ref: '#/components/headers/ContentVersion'
+        Location:
+          $ref: '#/components/headers/Location'
```

Comment:
❓ is it normal to put the status URI in the Location header for 201 responses? I would expect it to be in the body.

Reply:
@paouelle We return accepted, not created. https://restfulapi.net/http-status-202-accepted/ shows that the status URL is in the body, not the header.

Reply:
Now that you say it out loud like this, we should return created and not accepted. As the transform request was created and we are returning a URI to get its info.